middleware/validation.js

```diff
@@ -141,3 +141,80 @@ export const validatePassword = [
     .custom(value => !/\s/.test(value))
     .withMessage('No spaces are allowed in the password.')
 ];
+export const validateCreateComment = [
+  check('comment')
+    .exists()
+    .withMessage('Please include a comment in the body of request.')
+    .isString()
+    .trim()
+    .withMessage('Comment must be alphanumeric characters, please remove leading and trailing whitespaces.')
+    .isLength({ min: 2 })
```

Comment:
@olajide1234 did you guys agree on a min length for this?

Reply:
Yes, we have agreed min length for comments should be 2 characters.